draft-pda-protocol.md

```diff
@@ -1282,6 +1310,11 @@ the input shares and defeat privacy.
 
 # IANA Considerations
 
-TODO
+## Upload Extension Registry
+
+This document requests creation of a new registry for extensions to the Upload
+protocol. This registry should contain the following columns:
+
+[TODO: define how we want to structure this registry when the time comes]
```

Comment:
Do we need to provide this much structure for the extensions? IIUC at the PDA protocol layer, we will never examine or modify these extensions so why can't they be completely opaque blobs?

Reply:
This just creates the registry where extensions will go. We don't need to do anything more here, unless we decide we want to introduce some mandatory extensions.

Reply:
Well, my point applies more to the `struct Extension` definition. The extension just be an opaque blob of bytes, and the responsibility of protocol implementations would be to relay the blob unmodified from `PDAReport` to `PDAAggregateSubReq`. Then the format of that blob is something to be agreed upon by the client that generated it and the aggregator that is examining it, out of band from the PDA protocol.

Reply:
The contents of the extension, and how they're used, are up to the thing that defines the extension. So, yeah, I think what we have here aligns with what you envision.